sematic/resolvers/tests/test_silent_resolver.py

```diff
@@ -298,3 +320,30 @@ def test_deactivation_failures_for_resource():
     stored = resources[0]
 
     assert stored.status.state == ResourceState.ACTIVE
+
+
+@pytest.mark.skipif(
+    not RUN_SLOW_TESTS,
+    reason="This test takes a long time to execute, and is disabled by default",
```

Comment:
Is there a Python library to accelerate time? Probably not.

Reply:
Even if there were, our timeout logic uses `signal.alarm`, which makes a syscall. There's no way to speed those up.